schema/ChallengeSchema.js

```
@@ -39,7 +42,12 @@ const Challenge = mongoose.model('Challenge', challengeSchema)
 
 
 module.exports = {
-  addChallenge: (data) => new Challenge(data).save(),
+  addChallenge: (data) => {
+    if (!isProfane(data.description)) {
+      return new Challenge(data).save();
+    }
+    throw new Error('Unauthorized words have been sent');
+  },
```

I personally prefer to throw the error inside the if statement 😛

---

ok i will change that